castor/webhooks/tasks.py

```diff
@@ -0,0 +1,58 @@
+from datetime import datetime
+
+import requests
+from celery import shared_task
+
+from docker_events.models import DockerEvent
+from webhooks.models import Delivery
+from webhooks.models import WebHook
+
+
+@shared_task
+def dispatch_docker_event(docker_event_id):
+    """
+    Dispatch the DockerEvent identified by the given ID, to all WebHooks
+    subscribed to the DockerEvent's server events.
+    """
+    docker_event = DockerEvent.objects.get(id=docker_event_id)
+    webhooks = WebHook.objects.filter(docker_server=docker_event.docker_server)
+
+    for webhook in webhooks:
```

Comment:
I would break the dispatching of separate webhooks to different tasks - this way, if a webhook is slow will slow down the others too.